app/graphql/types/provider_enum.rb

```diff
@@ -1,6 +1,6 @@
 module Types
   class ProviderEnum < Types::BaseEnum
-    ::ComputeResource.supported_providers.keys.each do |provider|
+    ::ComputeResource.all_providers.keys.each do |provider|
       value provider
     end
```

Comment:
by defining enum values as a method, there would be no need to reload the class in tests

```suggestion
    def self.enum_values(context = {})
      ::ComputeResource.all_providers.keys.map do |provider|
        GraphQL::Schema::EnumValue.new(provider, owner: self)
      end
    end
```

Reply:
That would create new `EnumValue` objects every time the `Enum` is used? Seems like a waste to me, but I don't know how much of an actual problem that would be.

Reply:
Yes, this will create new `EnumValue` objects and is used for [dynamic enum values](https://graphql-ruby.org/schema/dynamic_types.html#using-enum_valuescontext). I don't think it will cause any problems.

Reply:
Cool. Thanks for the suggestion and explanation!

Updated.